deployment/utility/ssh.py

```diff
@@ -26,7 +27,7 @@ class OpenPaiSSH:
     def __init__(self, command, cluster_config_path, filter):
         self.cmd = command
         objectModelFactoryHandler = cluster_object_model.cluster_object_model(configuration_path=cluster_config_path)
-        self.cluster_object_model = objectModelFactoryHandler.run()
+        self.origin_machine_list = objectModelFactoryHandler.run()["machine"]["machine-list"]
```

Comment:
Hi @ydye,

Noticed the changes, and there could be one more step move forward:
passing the `machine_list` instead of `cluster_config_path`.

Consideration: here only need a `machine_list`, don't have to know it generate from `cluster_config_path` using a `objectModelFactoryHandler`.

Reply:
I think put this logic to paictl.py is not a good idea. Because we should make paictl more easy to read. And the logic should be separated into the class implementation module. And paictl only pass the raw parameters from the input.

Reply:
Done